docs/en/modules/tmr.md

```diff
@@ -9,23 +9,17 @@ It is aimed at setting up regularly occurring tasks, timing out operations, and
 
 What the tmr module is *not* however, is a time keeping module. While most timeouts are expressed in milliseconds or even microseconds, the accuracy is limited and compounding errors would lead to rather inaccurate time keeping. Consider using the [rtctime](rtctime.md) module for "wall clock" time.
 
-NodeMCU provides 7 static timers, numbered 0-6, and dynamic timer creation function [`tmr.create()`](#tmrcreate).
-
-!!! attention
-
-    Static timers are deprecated and will be removed later. Use the OO API initiated with [`tmr.create()`](#tmrcreate).
```

Comment:
I would keep this admonition but change the content. Stating that after a two year deprecation period static timers were finally removed in Q1 2019 or so.